src/renderer.zig

```diff
@@ -132,6 +135,9 @@ pub fn render(playerPosition: Vec3d) void {
 		const skyColor = vec.xyz(world.clearColor);
 		game.fog.skyColor = skyColor;
 
+		const fov: f32 = settings.fov + (game.Player.getSpeedModifierBlocking() * 20);
+		updateFOV(fov);
```

Comment:
Same issue as before: This function is called by the menu background recording below in line 567, so you must not change the fov in here.

Reply:
Ah, you're right.. I missed that call. Thinking of another way of doing it